bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -89,6 +146,141 @@
+		<description>Current Fan Mode</description>
+		<state readOnly="false">
+			<options>
+				<option value="auto">Auto</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanModeRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Fan Mode (Raw)</label>
+		<description>Current Fan Mode, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanState">
+		<item-type>String</item-type>
+		<label>Fan State</label>
+		<description>Current Fan State</description>
+		<state readOnly="true">
+			<options>
+				<option value="off">Off</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanStateRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Fan State (Raw)</label>
+		<description>Current Fan State, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="scheduleMode">
+		<item-type>String</item-type>
+		<label>Schedule Mode</label>
+		<description>Current Schedule Mode</description>
+		<state readOnly="false">
+			<options>
+				<option value="disabled">Disabled</option>
+				<option value="enabled">Enabled</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="scheduleModeRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Schedule Mode (Raw)</label>
+		<description>Current Schedule Mode, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="schedulePart">
+		<item-type>String</item-type>
+		<label>Schedule Part</label>
+		<description>Current Schedule Part</description>
+		<state readOnly="true">
+			<options>
+				<option value="morning">Morning</option>
+				<option value="day">Day</option>
+				<option value="evening">Evening</option>
+				<option value="night">Night</option>
+				<option value="inactive">Inactive</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="schedulePartRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Schedule Part (Raw)</label>
+		<description>Current Schedule Part, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timestampRuntime" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Runtime Record Timestamp</label>
+		<description>Time stamp of Runtime Update (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM"/>
+	</channel-type>
+
+	<channel-type id="heat1Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Heat1 Runtime Minutes</label>
+		<description>Run time in heat1 mode in minutes (Day 0 = TODAY, Day 6 = 6 days ago)</description>
+		<state readOnly="true" pattern="%d mins"/>
+	</channel-type>
+
+	<channel-type id="heat2Runtime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Why using `Number:Dimensionless` ?
You are providing a DecimalType when updating the channel, so `Number` looks appropriate.

Reply:
OK